OpenEdXMobile/src/test/java/org/edx/mobile/view/WebViewActivityTest.java

```diff
@@ -43,8 +42,8 @@ public void test_StartWebViewActivity_LoadsUrlAndShowsTitle()
      * @param url   The url to load
      * @param title The title to show, if any
      */
-    protected static void test_StartWebViewActivity_LoadsUrlAndShowsTitle(@NonNull String url,
-                                                                          @Nullable String title)
+    private static void test_StartWebViewActivity_LoadsUrlAndShowsTitle(@NonNull String url,
```

Comment:
I would make this non-static, since it's performing actual tests, and the parameters can later be injected by JUnit when the test is made parameterized.